### ISSUE TITLE
multiline size state

### ISSUE THREAD

**Issue by luoqisheng** [tbaranes/FittableFontLabel#39]
`  private func multiLineSizeState(rect: CGRect, size: CGSize) -> FontSizeState {
        // if rect within 10 of size
        if rect.height < size.height + 10 &&
           rect.height > size.height - 10 &&
           rect.width > size.width + 10 &&
           rect.width < size.width - 10 {
            return .fit
        } else if rect.height > size.height || rect.width > size.width {
            return .tooBig
        } else {
            return .tooSmall
        }
    }`

Hi, rect.width > size.width + 10 && rect.width < size.width - 10, Is there any mistake here?

**Comment by Minimistro32** [comment]
Not necessarily, the likelihood that the binary search lands on an exact value is very low. The range ±10 ensures that the binary search will return if it finds a valid value. Are you having an issue with it?
